tests/e2e/sanitizer_test.go

```diff
@@ -0,0 +1,265 @@
+package e2e
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"time"
+
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/tebeka/selenium"
+)
+
+var contextualReplacement = "RANDOM"
+
+var rulesFile = "rules/har.yaml"
+
+var sanitizeFileToExpectedSanitation = make(map[string][][]string)
+var changedLines = map[string]map[int]string{}
+
+/**
+Example formats
+sanitizeFileToExpectedSanitation = {
+	fileToSanitizeName1: [lineBeforeSanitization1, lineAfterSanitization1], [lineBeforeSanitization2, lineAfterSanitization2], ...],
+	fileToSanitizeName2: [[...], ...],
+	...
+}
+
+changedLines = {
+	sanitizedFileName1: {
+		lineNum1: lineAfterSanitization1,
+		lineNum2: lineAfterSanitization2,
+		...
+	},
+	sanitizedFileName2: {...},
+}
+*/
+
+func prepareMapTestScenario() {
+	sanitizeFileToExpectedSanitation["github.com.har"] = [][]string{{"password12345", "<REMOVED>"}}
```

Comment:
Instead of keeping track of lines changes, you can also add the expected sanitized files and do a diff with the original files to match with the visual output, and ensure that the expected sanitized file and the actual sanitized file have no diff.